### PR TITLE
Convert `covid19_tracking` dataset columns to use snakecase

### PR DESCRIPTION
**Pull request by adlersantos**
## Description

Addressing feedback from an internal peer review of the `covid19_tracking` pipelines:

* Standardize all BQ columns to use snakecase and lowercase

## Checklist
- [x] Tests pass.
- [x] Linters pass.
- [x] Please merge this PR for me once it is approved.
- [x] If this PR adds/edits a dataset or pipeline, it was reviewed and approved by the Google Cloud Public Datasets team beforehand.
- [x] If this PR adds/edits a dataset or pipeline, I put all my code inside  `datasets/<YOUR-DATASET>` and nothing outside of that directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ Every dataset and pipeline folder must contain a `dataset.yaml` and a `pipeline.
 
 # Best Practices
 
+- When naming BigQuery columns, always use snakecase and lowercase.
```
Comment (tswast) [inline README.md:193]: Let's use `snake_case` as an example :-)

```suggestion
- When naming BigQuery columns, always use `snake_case` and lowercase.
```

File: samples/pipeline.yaml [hunk 2]
```diff
@@ -117,7 +117,8 @@ dag:
         write_disposition: "WRITE_TRUNCATE"
 
         # The BigQuery table schema based on the CSV file. For more info, see
-        # https://cloud.google.com/bigquery/docs/schemas
+        # https://cloud.google.com/bigquery/docs/schemas.
+        # Always use snakecase and lowercase for column names.
```
Comment (tswast) [inline samples/pipeline.yaml:121]:
```suggestion
        # Always use snake_case and lowercase for column names.
```

**Review by tswast — APPROVED** [review]
A suggestion to make the docs slightly clearer

**Comment by adlersantos** [comment]
> A suggestion to make the docs slightly clearer

Thanks! I just applied your suggestions, hope you can check again once checks pass.

**Pull request merged**